src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -127,6 +128,7 @@ public async Task WriteTests(int[][] writes, WriteType writeType)
 
                     stream.Shutdown();
                     await stream.ShutdownWriteCompleted();
```

Comment:
is awaiting shutdown write completed even necessary?

Reply:
No, it is not. Removed it.